lib/src/checks/context_type.dart

```diff
@@ -1,3 +1,17 @@
+//  Copyright 2021 Abitofevrything and others.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
```

Comment:
And existing license in repo root should be updated since its missing some info https://github.com/nyxx-discord/nyxx_commands/blob/next/LICENSE#L189

Reply:
The root license file is intended to have the raw brackets (reading the paragraph above). The instructions also stipulate to include a notice in each file covered by the license.

Reply:
I think we talked about this before. Yeah you can do that but its not needed since license applied to disctribution of source code so whole repo applies to every file if not stated otherwise. https://infra.apache.org/apply-license.html#copy-per-file

So for me including license in each file just to include license in each file its not needed and only clutters file structure

Reply:
if your are talkign about NOTICE file https://infra.apache.org/apply-license.html#copy-per-file
you can just add NOTICE file to root directory of repo and you are covered but without that file license applies regardless

Reply:
I will remove the license notice from the individual files then, and leave the root file unchanged.